convert2rhel/backup/packages.py

```diff
@@ -73,6 +74,145 @@
 )
 _UBI_9_REPO_PATH = os.path.join(_RHSM_TMP_DIR, "ubi_9.repo")
 
+# NOTE: Over time we want to replace this with pkghandler.RestorablePackageSet.
+class RestorablePackage(RestorableChange):
+    def __init__(self, pkgs, reposdir=None, set_releasever=False, custom_releasever=None, varsdir=None):
+        """Keep control of installed/removed RPM pkgs for backup/restore.
```

Comment:
Is this comment right? This is used mainly when we are removing some packages from the system and during rollback we need to restore them.

From the comment I have a feeling I can use it also for installing some package during conversion and during rollback it will be removed.

I would go with "Keep control of removed RPM pkgs to be installed during rollback" if my understanding is right.

Reply:
modified the docstrings